### ISSUE TITLE
Bounce Not Working With My RIG

### ISSUE THREAD

**Issue by KingFeesh** [shteeve3d/blender-wiggle-2#15]
Hi, Currently using a rig for an online game and i am unsure why the volume bones  do not get the physics.

In the previous version of the wiggle bones i used the volume bones ( butt bone / pelvis /pecs ) to create physics or the animation.

In this version once using the same method of applying the Add on to the volume bone. Nothing happens.

Any ideas?

**Comment by shteeve3d** [comment]
from your video, it doesn't look like there's any obvious explanations for the issue. is it that specific bone or scene that's having issues, or are you not able to get 2.0 version of wiggle bones working at all?

**Comment by KingFeesh** [comment]
Hello,

The 2.0 version does not work at all.. once applied and following the instructions .

**Comment by shteeve3d** [comment]
i made a test setup that roughly resembles yours. if you play the timeline it should demonstrate the wiggle effect in action. i'm using the latest wiggle release 2.2.2 found here:
https://github.com/shteeve3d/blender-wiggle-2/releases/tag/v2.2.2

i created the file in blender 3.5.1, however i also tested as far back as 3.0 and still appears to open and play back fine. 
https://drive.google.com/file/d/1DaTqQn_zG3tJpmMnugwCXsQqpqpLmhFD/view?usp=sharing

if that test file works but your own scenes don't, then likely i'd need an example file from you demonstrating whatever setup is causing the issue in your scene. if the test file doesn't work, then it might be some issue with the addon installation itself (or conflict with another addon?) perhaps deleting and reinstalling the wiggle addon might be something to try.

**Comment by KingFeesh** [comment]
Ok great so I do see it in blender 3.0 ( the test you sent ). can you briefly explain what each action does . Mass , Stiff , Stretch , Damp + Gravity .

**Comment by shteeve3d** [comment]
There should be hover-able tooltips on most buttons to give a general idea of their behaviour.

But for your needs, assuming the addon is otherwise working correctly, the stiffness coefficient is the most relevant value for getting some wiggle. Lowering the tail stiffness to zero should make the bone hang purely under the influence of gravity (also note the gravity value in bones is a multiplier for blender’s global gravity in the world settings.

mass currently wouldn’t make much difference on single wiggling bones. It’s mostly for chains where you want certain segments to be heavier than others.

**Comment by KingFeesh** [comment]
Tried the 2.2.2 version. can i send you the blender file for my animation?

**Comment by shteeve3d** [comment]
Yeah whatever you’re comfortable sending, would be helpful to diagnose.
